Optimize (CQA) - Mng content - Lifecycles (#4867)
theforeman/foreman-documentation: guides/common/modules/proc_removing-lifecycle-environments-from-smart-proxy-by-using-cli.adoc

= Removing lifecycle environments from {SmartProxyServer} by using Hammer CLI

[role="_abstract"]

> Remove lifecycle environments from {SmartProxyServer} when they are obsolete or were added incorrectly, then resynchronize so the {SmartProxy} only stores content your hosts still need.

Reviewer: I would consider splitting this rather long sentence. Not a hard requirement.

```suggestion
Remove lifecycle environments from {SmartProxyServer} when they are obsolete or were added incorrectly, then resynchronize. That way, the {SmartProxy} only stores content your hosts still need.
```

Reviewer 2: Please also consider "the {SmartProxy}" -> "your {SmartProxyServer}".

Author: ```suggestion
Remove lifecycle environments from {SmartProxyServer} when they are obsolete or were added incorrectly, then resynchronize.
That way, your {SmartProxyServer} only stores content your hosts still need.
```
